toolset/setup/linux/systools/ant.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+RETCODE=$(fw_exists ${IROOT}/ant.installed)
+[ ! "$RETCODE" == 0 ] || { \
+source $IROOT/ant.installed
+return 0; }
+
+sudo apt-get update
+sudo apt-get -y --force-yes install ant
+
+if [ -e /usr/bin/ant ]
+then
+    sudo rm -f /usr/bin/ant
+fi
+sudo ln -s /usr/share/ant/bin/ant /usr/bin/ant
```

Comment:
If so, I think maybe get rid of `/usr/bin/ant` entirely and just replace this line with `echo "export PATH=/usr/share/ant/bin:\$PATH" > $IROOT/ant.installed`